src/core/schema/config.model.schema

```diff
@@ -358,6 +358,30 @@
           "title": "Log to browser console?"
         }
       }
+    },
+
+    "_scrollingContainer": {
+      "type": "object",
+      "title": "iFrame and Screen Reader scrolling support",
+      "properties" : {
+        "_isEnabled": {
+          "type": "boolean",
+          "default": false,
+          "required": true,
+          "inputType": "Checkbox",
+          "validators": [],
+          "title": "Enabled?"
+        },
+        "_limitToSelector": {
+          "type": "string",
+          "required": false,
+          "default": "",
+          "title": "Limit to selector",
+          "inputType": "Text",
+          "validators": [],
+          "help": "A comma-separated list of classes to look for on the HTML element when determining whether this should be active or not."
```

Comment:
Please could we encourage using the selector first and foremost? (I know it now does `.hasClass()` behind the scenes too but that, and the help text, is a hangover from the older implementation).